pom.xml

```diff
@@ -57,7 +57,7 @@
         <spring.boot.version>2.5.6</spring.boot.version>
         <java.version>1.8</java.version>
         <logback.version>1.2.3</logback.version>
-        <hadoop.version>2.7.3</hadoop.version>
+        <hadoop.version>2.8.1</hadoop.version>
```

Comment:
Is it necessary to upgrade this hadoop version ?
To upgrade the jar package, you need to modify both license and note.

Reply:
The problem is in the org.apache.hadoop:hadoop-ws jar.
Hadoop has officially solved this problem, but it has only been submitted to versions 2.8.x and 3.0.0-alpha1. This is the  [link](https://issues.apache.org/jira/browse/HADOOP-12963)

In order to solve this problem, our company has upgraded hadoop to 2.8.5。
I will modify both license and note later.